filter/series_by_tag.go

```diff
@@ -34,13 +37,15 @@ type TagSpec struct {
 	Value    string
 }
 
-func transformWildcardToRegexpInSeriesByTag(input string) string {
-	var result = input
-	var re = regexp.MustCompile(`\{(.*?)\}`)
-	var correctLengthOfMatchedWildcardIndexesSlice = 4
+func transformWildcardToRegexpInSeriesByTag(input string) (string, bool) {
+	var (
+		result                                     = input
```

Comment:
I think it would be more ideomatic to use `:=` instead of the var bloc, if it is possible
```suggestion
		result := input
```